.github/workflows/CI.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches:
+      - Wojtek
+      - Slawek
+      - Pati
+  workflow_dispatch:
```

Comment:
po co to ustawienie skoro jest puste ?

Reply:
Ono nie jest puste, pod spodem masz caly workflow haha